library/core/src/marker.rs

```diff
@@ -879,25 +879,35 @@ marker_impls! {
         {T: ?Sized} &mut T,
 }
 
-/// Types that can be safely moved after being pinned.
+/// Types that do not need to follow the rules of pinning.
```

Comment:
"do not (need) to follow the rules" makes it sound like these are the complicated types, the weird ones that don't follow the rules, rather than actually being the default for most types.

Maybe it's better to word it as types that don't have pinning restrictions, or something like that.

Reply:
True, I was not quite happy with this wording either. Will revise..

Reply:
That was the intention of the new wording but I can see the other perspective too. Maybe,

"Types that can ignore pinning-related restrictions"?

Reply:
> Your comment is interesting, because I had the opposite reaction! For me, the rules of pinning are quite complicated, so a type that "does not need to follow the rules of pinning" must therefore follow the comparatively more simple rules that govern all other types.

Ah, that makes sense.

Maybe something like *"Types that aren't affected by pinning restrictions."* could work.

I don't feel strongly about this, so feel free to just ignore my comment if you're happy with the current wording. :)